addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1062,6 +1105,8 @@ def forward_member_field(field_name):
             "last_interest_dt",
             "member_count",
             "name",
+            "is_self_channel_owner",
+            "is_self_channel_admin",
```

Comment:
These fields are not necessary. They should be on members not on channel.
The current user should never broadcast "is_self_channel_owner" for the channel to other users.